src/course/syllabus/application/fruit-shop/fruit-shop.json

```diff
@@ -0,0 +1,6 @@
+{
+  "tabs": {
+    "schedule": false,
+    "spikes": false
+  }
+}
```

Comment:
I probably should have said that you can put this in the directory above to avoid having to repeat the same JSON for every single project. E.g. put a file named `application.json` inside the `/syllabus/application` folder and it'll apply to every page within

Reply:
Ah, nice. I'm gonna tidy this branch up sometime tomorrow. Do leave other comments if you have time - but equally, enjoy your evening!